### DISCUSSION TITLE
Missing Sensor 01 - ENTITY from main page

### DISCUSSION THREAD

**Discussion by feinadam** [Blackymas/NSPanel_HA_Blueprint#1651] [General]
Does anyone else have "Sensor 01 - ENTITY (Optional)" missing from the main page?
Unfortunately I have uploaded together the 4.2.2 and the idf lib transition. Therefore I don't know wich one is cousing the problem.

I have tried to display several entity on Sensor 01 but none of them was displayed. 

> I have two error between the log:
> Logger: homeassistant.components.automation.nspanel_configuration
> Source: components/automation/__init__.py:690
> Integration: Automatizmus (documentation, issues)
> First occurred: 15:41:47 (1 occurrences)
> Last logged: 15:41:47
> 
> Error while executing automation automation.nspanel_configuration: TypeError: argument of type 'NoneType' is not iterable

And another one:

> Logger: homeassistant.components.automation.nspanel_configuration
> Source: helpers/script.py:1792
> Integration: Automatizmus (documentation, issues)
> First occurred: 15:41:47 (5 occurrences)
> Last logged: 15:41:47
> 
> NSPanel Configuration: Main choices: NSPanel event: Choose at step 2: Page changed: Choose at step 1: Home page: If at step 7: Error executing script. Error rendering template for variables at pos 2: TypeError: argument of type 'NoneType' is not iterable
> NSPanel Configuration: Main choices: NSPanel event: Choose at step 2: Page changed: Choose at step 1: Home page: Error executing script. Error rendering template for if at pos 7: TypeError: argument of type 'NoneType' is not iterable
> NSPanel Configuration: Main choices: NSPanel event: Choose at step 2: Page changed: Error executing script. Error rendering template for choose at pos 1: TypeError: argument of type 'NoneType' is not iterable
> NSPanel Configuration: Main choices: NSPanel event: Error executing script. Error rendering template for choose at pos 2: TypeError: argument of type 'NoneType' is not iterable
> NSPanel Configuration: Error executing script. Error rendering template for choose at pos 2: TypeError: argument of type 'NoneType' is not iterable
> 
Many thanks!

**Comment by feinadam** [comment]
I have identified the isse above is due to the "Climate to control - ENTITY (Optional)". The climate page seems not wokring, buttons are not loading etc.
When I have removed the climate entity, the Sensor 01 - ENTITY appeas on the main screen.

**Reply by edwardtfn** [reply]
We have just release v4.2.4 with some fixes around climate entities. If you still having this issue after updating to v4.2.4, could you please report it as a bug in the [Issues](https://github.com/Blackymas/NSPanel_HA_Blueprint/issues) area?

**Reply by feinadam** [reply]
With 4.2.3 and 4.2.4 the issue has been solved. Thank you!

**Comment by edwardtfn** [comment]
Could you please report this as a bug? This is the link: https://github.com/Blackymas/NSPanel_HA_Blueprint/issues/new/choose

It would be nice if you can share the attributes (developer tools > States) from that entity related to the crash.
